homepage.css

```diff
@@ -1,3 +1,7 @@
+/* Import  font from google fonts*/
```

Comment:
26/8/2021: I import font from google fonts. The fonts I choice is Source sans pro and Open sans